lib/cdo/sauce_connect.rb

```diff
@@ -52,9 +54,9 @@ def start_sauce_connect(daemonize: false)
         tests_started, log_lines = tests_started?(log_file)
 
         if tests_started == :success
-          log "SUCCESS, sc is running#{" in the background, you can stop it with `killall sc`" if daemonize}"
+          log "SUCCESS, sc is running in the background, you can stop it with `killall sc`"
 
-          at_exit {stop_sauce_connect} unless daemonize
```

Comment:
Can you clarify why it's no longer necessary for us to stop the process in this case?

Reply:
This code path was dead because we always specify `daemonize: true`, so this is just removing dead code.